### PR TITLE
[#439, #444, #409] - Fixes

### PR DESCRIPTION
**Pull request by Kasmadei**
Fixes for: 

1) https://github.com/kbss-cvut/fta-fmea-ui/issues/439
2) https://github.com/kbss-cvut/fta-fmea-ui/issues/444
3) https://github.com/kbss-cvut/fta-fmea-ui/issues/409

### CODE REVIEW

**Review by blcham — APPROVED** [review]
LGTM, although do not like that it is not separated in commits

**Review by LaChope — APPROVED** [review]
Just see minor comment, LGTM

**Comment by Kasmadei** [comment]
@blcham updated

File: src/components/dashboard/content/list/FaultTreeOverview.tsx [hunk 1]
```diff
@@ -26,6 +27,7 @@ const FaultTreeOverview = () => {
   const [contextMenuAnchor, setContextMenuAnchor] = useState<ElementContextMenuAnchor>(contextMenuDefaultAnchor);
   const [createFaultTreeDialogOpen, setCreateFaultTreeDialogOpen] = useState<boolean>(false);
   const [selectedSystem] = useSelectedSystemSummaries();
+  const isTreeCreateDisabled = !selectedSystem;
```
[the remark this answers is not in the document]
Comment (Kasmadei) [reply src/components/dashboard/content/list/FaultTreeOverview.tsx]: Sure, renamed

**Review by blcham — APPROVED** [review]
LGTM

**Pull request merged**